cloudberry/zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/TwitterDataStoreForPostgreSQL.scala

```diff
@@ -0,0 +1,49 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+
+import edu.uci.ics.cloudberry.zion.model.schema._
+
+object TwitterDataStoreForPostgreSQL {
```

Comment:
I think the naming cause the confusion. I would call it `TwitterDataStoreWithHashTag` and the other one is `TwitterDataStoreWithoutHashTag`. They are two different table that can be used to both db.

Reply:
Ok.